### ISSUE TITLE
Single tabs dont get listed.

### ISSUE THREAD

**Issue by mahoromax** [cnwangjie/better-onetab#180]
扩展版本 (extension version): 1.4.4
浏览器版本 (browser version): Chrome Version 74.0.3729.131 (Official Build) (64-Bit)
操作系统 (operating system): Windows 10 
问题描述 (problem description): If there's only a single tab open in a window, both the list option "store all tabs in current window" or the option for the same behaviour of the addon button are not working as expected. The addon tab is opening, but the previous open tab does not get listed nor closed.
After the "better onetab" tab is open,  the button still doesnt work, but the list option does work as intended, leaving me with only the "better onetab" tab opened (and the original tab listed).

期望行为 (expected behavior): Single tab is listed in onetab and closed.

**Comment by cnwangjie** [comment]
Is the only tab is newtab? If so please try it with a common website (url start with http:// or https://). The tab with an illegal URL will be ignored by default.

Besides, it is a bug fixed in v1.4.5 that no response when the icon is clicked.

**Comment by mahoromax** [comment]
No, I tried it with different tabs, like a google search or even this github issue.
